src/components/Youtube.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export default ({ className, src }) => {
+  const youtubeID = new URL(src).searchParams.get('v')
+  const embedUrl = `https://youtube.com/embed/${youtubeID}`
+
+  return (
+    <iframe
+      title="YouTube Video"
+      id="ytplayer"
+      type="text/html"
+      className={className}
+      src={embedUrl}
+      frameborder="0"
+      allowfullscreen
+    />
+  )
```

Comment:
thoughts on stylizing this using https://developers.google.com/youtube/player_parameters#IFrame_Player_API instead, will give us more flexibility in stylization

e.g.
* modestbranding=1

Reply:
Are there any benefits to this over just appending the options to the src? ie `https://youtube.com/embed/${youtubeID}?modestbranding=1`

Reply:
wait didnt know you could do that lmao ok nice